ql/termstructures/yield/fittedbonddiscountcurve.hpp

```diff
@@ -323,6 +329,11 @@ namespace QuantLib {
         return optimizationMethod_;
     }
 
+    inline Constraint
```

Comment:
I think it would be better to return `const Constraint&` to avoid a copy where not needed. Same with the other methods in this class in a future PR.

Reply:
Yeah, that would be C++ 1998.
Not gonna do that.

Reply:
It's reasonably done so, smartass.

Reply:
It's like... oh wait... Sweemer is out there. He has never done anything before. But for sure he is making sure this is gonna done right. :)

Reply:
Fuck off!

Reply:
Cay, I have to ask—are you going to be a problem for this project?  Because whatever someone's contributing is not worth causing other people to leave because they're insulted.

(Also, a word to the wise: even if you delete or edit comments, they are also sent via email to me and whoever commented on the PR.)

Reply:
Just let me know, what is wrong.

Reply:
> (Also, a word to the wise: even if you delete or edit comments, they are also sent via email to me and whoever commented on the PR.)

And also to everyone who turned on `All Activity` in the notifications.